src/shared/utils/index.ts

```diff
@@ -138,3 +139,31 @@ export const labelOf = (string: string) => {
 };
 
 export const isBrowser = typeof window !== 'undefined';
+
+/**
+ * Returns the logout URL of the realm the user is authenticated with
+ *
+ * @param identities
+ * @param realms
+ */
+export function getLogoutUrl(
+  identities: Identity[],
+  realms: { label: string; endSessionEndpoint: string }[]
+): string {
+  // find authenticated Identity and get realm name
+  const auths = identities.filter(
```

Comment:
This operation could be a `.find()` returning either the matching identity or a realm, since you only care about the 1st match.

Reply:
yes! that's the one I was looking for. For some reason I keep confusing it with `.some()` which only returns a boolean

Reply:
fixed in 7172506e898cf880598ca2dfa8d90588445f222f